src/Sulu/Bundle/SecurityBundle/Resources/public/js/components/roles/edit/details/main.js

```diff
@@ -27,6 +30,7 @@ define(['config'], function(Config) {
 
         initialize: function() {
             this.saved = true;
+            this.options.data = this.options.data();
```

Comment:
`this.options` should actually be considered immutable, isn't there a better way to handle this?